### ISSUE TITLE
Prediction demo using different pretrained models

### ISSUE THREAD

**Issue by kshitijd20** [Sense-X/X-Temporal#16]
Hi,

Thanks for sharing the code and model zoo. I was wondering if you could direct me how should I modify this code to output predictions on single video files (e.g. .mp4 format) like it is done in slowfast code [here](https://github.com/facebookresearch/SlowFast/blob/master/VISUALIZATION_TOOLS.md)

Thanks.

**Comment by deepcs233** [comment]
To get the predicted logits, you can refer to this issue [https://github.com/Sense-X/X-Temporal/issues/9](https://github.com/Sense-X/X-Temporal/issues/9). For single video input, only need to write one line in meta file and set the falg `video_input` in `test.yaml`

**Comment by kshitijd20** [comment]
Thanks for the reply, it works
